enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -145,11 +150,12 @@ export const InteractiveQuestionProvider = ({
     modelsFilterList: mapEntityTypeFilterToDataPickerModels(entityTypeFilter),
     isSaveEnabled,
     saveToCollectionId,
+    isCardIdError,
   };
 
   useEffect(() => {
-    loadQuestion();
-  }, [loadQuestion]);
+    loadAndQueryQuestion();
```

Comment:
nothing changes, it's just a renaming to make it more clear what's being executed here.